ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -60,7 +60,7 @@ jobs:
       class: RunPytest
       args:
         build_url: '{fedora-30/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
+        test_suite: test_integration/test_installation.py::TestInstallReplicaAgainstSpecificServer
         template: *ci-master-f30
         timeout: 3600
         topology: *master_1repl_1client
```

Comment:
The topology should be master_2repl_1client otherwise the test won't be executed (it requires num_replicas = 2).